website/docs/docs/cloud/about-cloud/architecture.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "dbt Cloud Architecture"
+id: "architecture"
+description: "Information about the architecture, communication, and security of dbt Cloud"
+---
+
+This page is intended to help practitioners and those interested in the architecture and data flow of the hosted dbt Cloud product.
+
+## About dbt Cloud architecture
+
+The dbt Cloud application comprises a set of static and dynamic components. The static components are constantly running to serve highly available dbt Cloud functionality, for example, the dbt Cloud web application. The dynamic components are created ad-hoc to fill background jobs or requests to use the IDE. 
+
+dbt Cloud is available in most regions around the world in both [single tenant](/docs/cloud/about-cloud/tenancy#single-tenant)(AWS and Azure) and [multi-tenant](/docs/cloud/about-cloud/tenancy#multi-tenant) configurations.  
+
+dbt Cloud uses PostgreSQL for its backend, S3-compatible Object Storage systems for logs and artifacts, and a Kubernetes storage solution for creating dynamic, persistent volumes. 
+
+All data at rest on dbt Cloud servers is protected using AES-256 encryption. 
+
+<img src="/img/docs/dbt-cloud/on-premises/data-flows.png" />
+
+For a more detailed breakdown of the dbt Cloud apps, [download the advanced architecture guide PDF](https://drive.google.com/uc?export=download&id=1lktNuMZybXfqFtr24J8zAssEfoL9r51S).
```

Comment:
```suggestion
For a more detailed breakdown of the dbt Cloud application, [download the advanced architecture guide PDF](https://drive.google.com/uc?export=download&id=1lktNuMZybXfqFtr24J8zAssEfoL9r51S).
```

Reply:
This should say "apps" plural. Yes, the Cloud product can be called an "app" but it's made up of individual apps all doing different things on their own and together.